prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -749,10 +750,12 @@ def _retrieve_context(output, doc_id, vector_index, answer) -> str:
     nodes = retriever.retrieve(answer)
     text = ""
     for node in nodes:
-        if node.score > 0.6:
+        # ToDo: May have to fine-tune this value for node score or keep it
+        # configurable at the adapter level
+        if node.score > 0:
```

Comment:
@gaya3-zipstack Wouldn't recommend to do this by default.

If the current state of an adapter still warrants this change, then can we selectively do this for absolutely required adapters only, and keep the same 0.6 score threshold as default for others?

Reply:
Another suggestion might be to keep this configurable - either by env or by the user as a challenger setting if we are unsure of what the right one should be